### PR TITLE
optional branch input via GIT_REF

### PR DESCRIPTION
**Pull request by nordfogel**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -24,18 +24,22 @@ inputs:
...
+    description: "The contents of an SSH known_hosts file for host key verification"
     required: false
   GIT_SSH_NO_VERIFY_HOST:
-    description: "if true, skips host key verification. Mutually exclusive with GIT_SSH_KNOWN_HOSTS"
+    description: "If true, skip host key verification. Mutually exclusive with GIT_SSH_KNOWN_HOSTS"
     default: "false"
     required: false
+  GIT_REF:
+     description: "Push another ref rather than the current branch (GITHUB_REF) from which the action was triggered. Only effective when PUSH_ALL_REFS is false"
+     default: ${{ github.head_ref || github.ref_name }}
```
[the remark this answers is not in the document]
Comment (nordfogel) [reply action.yml:35]: My bad, you are absolutely right.

Comment (nordfogel) [inline action.yml:35]:
```suggestion
     default: ${{ github.ref }}
```

**Review by nordfogel — COMMENTED** [review]
Wrong default ref

**Comment by spyoungtech** [comment]
Looks good. Thanks for the contribution

**Pull request merged**